scalableminds/webknossos
# Unicode replacement

### Mailable description of changes:
 - Prevent users from entering Unicode chars and other special characters as their first or last name

- user schema got changed to match new rules of name creation

### Steps to test:
- enter an invalid first or last name, which can't be replaced
- get an error message
- if email is already in use get also an error message
- there should be multiple error messages in the right order 

### Issues:
- fixes #2114 

------
